scene/src/ui/Announcement.tsx

```diff
@@ -0,0 +1,33 @@
+import { type Color4 } from '@dcl/sdk/math'
+import ReactEcs, { Label } from '@dcl/sdk/react-ecs'
+
+type AnnouncementProps = {
+  visible: boolean
+  color: Color4
+  text: string
+}
+
+function Announcement({
```

Comment:
Replicate the logic of https://github.com/decentraland-scenes/Antrom-RPG/blob/main/scene/src/ui/announcement/announcement.tsx

Reply:
then with conditional rendering https://github.com/decentraland-scenes/Antrom-RPG/blob/main/scene/src/controllers/ui.controller.tsx#L88